contracts/tests/sbtc-bootstrap-signers.test.ts

```diff
@@ -0,0 +1,46 @@
+import {
+    alice,
+    signers,
+  } from "./helpers";
+  import { test, expect, describe } from "vitest";
+  import { txOk, txErr } from "@clarigen/test";
+  
+  describe("sBTC bootstrap signers contract", () => {
+    describe("Rotate keys tests", () => {
+      test("Rotate keys wrapper correctly", () => {
+        const receipt = txOk(
+          signers.rotateKeysWrapper({
+            newKeys: [new Uint8Array(32).fill(0),new Uint8Array(32).fill(0)],
+            multiSigAddress: alice,
+            newAggregatePubkey: new Uint8Array(32).fill(0),
```

Comment:
Yeah, now that I'm seeing that we'd also want JS code helpers for generating the multi-sig address, let's punt on automatically calculating it and open a ticket